src/composer.json

```diff
@@ -53,7 +54,7 @@
     },
     "config" : {
         "platform" : {
-            "php" : "5.6"
+            "php" : "7.0.8"
```

Comment:
Is it possible to make more general like `7.0`? Because not every distro on PHP 7.0 will have 7.0.8.

Reply:
updated